WebRoot/js/ajax/dwt/xforms/XModelItem.js

```diff
@@ -407,6 +410,7 @@ XModelItem.prototype.validateEmailAddress = function(value) {
 XModelItem.prototype._normalizeAndValidate = function(value) {
 
     var whiteSpace = this.getWhiteSpace();
+	var stringMismatchErrorMessage = this.getCustomErrorMessage() || "stringMismatch";
```

Comment:
* fix indent
* just declare the variable here
```suggestion
	var stringMismatchErrorMessage;
```